articles/multifactor-authentication/api/oob.md

```diff
@@ -1,14 +1,17 @@
 ---
-title: Associate an OOB Authenticator
-description: How to associate an OOB authenticator
+title: Associate an Out-of-Band Authenticator
+description: Configure your application so users can self-associate out-of-band (OOB) authenticators.
 ---
-# Associate an OOB Authenticator
 
-In this tutorial, we will show you how you can configure your Auth0 tenant to allow the self-association of out-of-band (OOB) authenticators.
+# Associate an Out-of-Band Authenticator
 
-## Step 1. Trigger the MFA Error and Use the MFA Token to Associate the New Authenticator
+In this tutorial, you'll learn how to configure your application so users can self-associate out-of-band (OOB) authenticators.
 
-Whenever a user begins the authorization process and they do not have an active authenticator associated with their account, they will trigger the following MFA response when calling the `/oauth/token` endpoint:
+<%= include('./_before-you-start.md') %>
```

Comment:
The current _before-you-start_ snippet is good for the Associate API, but doesn't work here. The tutorials that interact with the Token Endpoint should display the proper requirements which are similar to https://github.com/auth0/docs/pull/6035/files#diff-0a46c4ed6966526418bf5fc42c32d2b8R10. These requirements are more about the configuration required in the tenant.